com.unity.postprocessing/PostProcessing/Runtime/PostProcessRenderContext.cs

```diff
@@ -33,6 +33,12 @@ public Camera camera
                     stereoRenderingMode = StereoRenderingMode.SinglePass;
                     numberOfEyes = 1;
 
+                    if (XRSettings.stereoRenderingMode == XRSettings.StereoRenderingMode.SinglePassMultiview)
+                    {
+                        stereoRenderingMode = StereoRenderingMode.SinglePassMultiview;
```

Comment:
As discussed in your previous multiview PR, we don't support multiview feature for PPv2 and it currently doesn't work due to engine not providing input texture correctly (see[ discussion link](URL https://github.com/Unity-Technologies/Graphics/pull/5917#discussion_r723683891)).  
Thus, please remove all instances of multiview from this PR.